queries/deleteMovie.js

```diff
@@ -0,0 +1,10 @@
+import pool from "../database/db.js";
+
+const deleteMovie = async (id) => {
+  const deleteQuery = "delete from movies where MovieID = $1 returning *;";
+  const values = [id];
+  const res = await pool.query(deleteQuery, values);
+  return res.rows;
+};
```

Comment:
The convention that Robert & I like to follow is to only use arrow functions as callback functions. By using arrow functions for standalone functions, your eyes have to do more work to figure out that it's a function. The first few words look like we're creating a variable, so my eyes have to scan a lot more to realize it's a function. If you were to use the `function` keyword and create your functions that way, the first word my eyes would see is `function`, so I'd be able to read and figure out it's a function faster